src/main/java/uk/nhs/digital/nhsconnect/nhais/translator/AcceptanceBirthTranslator.java

```diff
@@ -1,18 +1,83 @@
 package uk.nhs.digital.nhsconnect.nhais.translator;
 
+import lombok.RequiredArgsConstructor;
 import org.hl7.fhir.r4.model.Parameters;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 import uk.nhs.digital.nhsconnect.nhais.exceptions.FhirValidationException;
+import uk.nhs.digital.nhsconnect.nhais.mapper.AcceptanceCodeMapper;
+import uk.nhs.digital.nhsconnect.nhais.mapper.AcceptanceDateMapper;
+import uk.nhs.digital.nhsconnect.nhais.mapper.AcceptanceTypeMapper;
+import uk.nhs.digital.nhsconnect.nhais.mapper.GpNameAndAddressMapper;
+import uk.nhs.digital.nhsconnect.nhais.mapper.PartyQualifierMapper;
+import uk.nhs.digital.nhsconnect.nhais.mapper.PersonAddressMapper;
+import uk.nhs.digital.nhsconnect.nhais.mapper.PersonDateOfBirthMapper;
+import uk.nhs.digital.nhsconnect.nhais.mapper.PersonNameMapper;
+import uk.nhs.digital.nhsconnect.nhais.mapper.PersonSexMapper;
+import uk.nhs.digital.nhsconnect.nhais.model.edifact.BeginningOfMessage;
+import uk.nhs.digital.nhsconnect.nhais.model.edifact.DateTimePeriod;
+import uk.nhs.digital.nhsconnect.nhais.model.edifact.ReferenceTransactionNumber;
+import uk.nhs.digital.nhsconnect.nhais.model.edifact.ReferenceTransactionType;
 import uk.nhs.digital.nhsconnect.nhais.model.edifact.Segment;
-import uk.nhs.digital.nhsconnect.nhais.service.EdifactToFhirService;
+import uk.nhs.digital.nhsconnect.nhais.model.edifact.SegmentGroup;
+import uk.nhs.digital.nhsconnect.nhais.service.TimestampService;
 
 import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static uk.nhs.digital.nhsconnect.nhais.mapper.FromFhirToEdifactMapper.emptyMapper;
 
 @Component
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
 public class AcceptanceBirthTranslator implements FhirToEdifactTranslator {
 
+    private final PartyQualifierMapper partyQualifierMapper;
+    private final GpNameAndAddressMapper gpNameAndAddressMapper;
+    private final AcceptanceCodeMapper acceptanceCodeMapper;
+    private final AcceptanceTypeMapper acceptanceTypeMapper;
+    private final AcceptanceDateMapper acceptanceDateMapper;
+    private final PersonNameMapper personNameMapper;
+    private final PersonSexMapper personSexMapper;
+    private final PersonAddressMapper personAddressMapper;
+    private final PersonDateOfBirthMapper personDateOfBirthMapper;
+    private final TimestampService timestampService;
+
     @Override
     public List<Segment> translate(Parameters parameters) throws FhirValidationException {
-        return null;
+
+        return Stream.of(
+            //BGM
+            emptyMapper(new BeginningOfMessage()),
+            //NAD+FHS
+            partyQualifierMapper,
+            //DTM+137
+            emptyMapper(new DateTimePeriod(timestampService.getCurrentTimestamp(), DateTimePeriod.TypeAndFormat.TRANSLATION_TIMESTAMP)),
```

Comment:
Don't provide a timestamp here. FhirToEdifactService will do it for you. Make sure we have the same timestamp in the header and the body.